### ISSUE TITLE
Align GUI on device edit page

### ISSUE THREAD

**Issue by moteus** [fusionpbx/fusionpbx#1669]
![screenshot_1](https://cloud.githubusercontent.com/assets/1804306/16121485/28a6a808-33ec-11e6-9d5b-67f45e35089d.png)

**Comment by moteus** [comment]
Similar problem on Log viewver page
![screenshot_2](https://cloud.githubusercontent.com/assets/1804306/16150276/e117a7e0-349e-11e6-8290-530823a143e2.png)

**Comment by reliberate** [comment]
This is actually not really a _problem_ in the usual sense of the word.  It's a side effect that will occur on _any_ page that contains really wide content, due to the integration of Bootstrap.  Eventually, each page within FusionPBX will be updated to be responsive (using Bootstrap), and this _problem_ will go away.  However, in the meantime, it is noticeable on a few different pages, particularly those with really wide content that can't currently, and automatically, adjust itself to the size of it's container/window.

In other words, these will get remedied... but will take some time.
